pr_watch/models.py

```diff
@@ -53,11 +56,13 @@ class WatchedFork(models.Model):
     # uses internal id key
     enabled = models.BooleanField(default=True)
     # This is the old .env variable WATCH_ORGANIZATION
-    organization = models.CharField(
-        max_length=200,
+    organization = models.ForeignKey(
+        Organization,
+        on_delete=models.CASCADE,
         db_index=True,
-        help_text=("Watched GitHub organization. E.g.: open-craft. PRs against the watched fork "
-                   "made by members of this organization will trigger a sandbox build"),
+        help_text=(
+            "Organization to watch. PRs against against the watched fork"
```

Comment:
@rocioar But now this is missing the second part of the sentence... To be completely clear, here's what I think this help text should say:

> Organization to watch. PRs against the watched fork made by members of this organization will trigger a sandbox build.